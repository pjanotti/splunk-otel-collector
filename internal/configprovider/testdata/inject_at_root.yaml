config_sources:
  tstcfgsrc:

__root__:  ${tstcfgsrc:valid_yaml_string}
