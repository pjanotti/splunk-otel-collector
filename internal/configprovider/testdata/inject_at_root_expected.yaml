bool: true
int: 42
source: string
map:
  k0: v0
  k1: v1
